src/components/CreateNewScenarioModal/CreateNewScenarioModal.js

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import FileInputIcon from 'lucide-react/dist/esm/icons/file-input';
+import FileQuestionMarkIcon from 'lucide-react/dist/esm/icons/file-question';
+import LayersIcon from 'lucide-react/dist/esm/icons/layers';
```

Comment:
Do we usually import icons from "lucide-react/dist/esm"? (we should try to be consistent between the different files 😄)